web/spec/supabase.yml

```diff
@@ -903,7 +903,7 @@ pages:
           ```js
           const { data: user, error } = await supabase.auth.api.updateUserById({
```

Comment:
Thanks for picking this up.

Would you mind additionally removing the braces from lines 904 and 907? The underlying [gotrue](https://github.com/supabase/gotrue-js/blob/master/src/GoTrueApi.ts#L722) method expects an argument of `uid : '123', attributes: { key: 'value', .... }` instead of `{ uid : '123', attributes: { key: 'value', .... } }`.

Note this comment about argument shape also applies to the examples about updating a user's phone number, email address, and (app) metadata.

Reply:
Thank you for your kind reply. I should have checked the function, it's being passed as a parameter, not as a destructured object. If I remove the curly braces, then I have to remove these tags (uid:,attributes:) as well. Right?

Reply:
cc: @stuartkumar0630 @kiwicopple

Reply:
No problem. Yeah I think you're right about those tags, I should've removed those in my comment

Reply:
@stuartkumar0630 So, shall I remove those tags for all updateUserById examples and then update PR?

Reply:
The tags and one pair of braces will need to be removed from all the examples. So

`const { data: user, error } = await supabase.auth.api.updateUserById({
      uid: '6aa5d0d4-2a9f-4483-b6c8-0cf4c6c98ac4',
      attributes: { email: 'new@email.com' },
})` 

will become 

`const { data: user, error } = await supabase.auth.api.updateUserById(
     '6aa5d0d4-2a9f-4483-b6c8-0cf4c6c98ac4',
     { email: 'new@email.com' },
)`

Reply:
Made the proposed changes.